src/Ui.GraphiQL/Ui.GraphiQL.csproj

```diff
@@ -12,8 +12,12 @@
 	  <EmbeddedResource Include="Internal\graphiql.cshtml" />
 	</ItemGroup>
 
-	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="$(MicrosoftAspNetCoreHttpAbstractionsVersion)" />
-	</ItemGroup>
+  <ItemGroup Condition="'$(IsNetCore30OnwardsTarget)' == 'True'">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Documentation says that `Microsoft.AspNetCore.App` should be removed since ASP.NET Core now is a share framework - https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?tabs=visual-studio&view=aspnetcore-2.2#remove-obsolete-package-references

Reply:
But it seems that it is only for Microsoft.NET.Sdk.Web SDK.

Reply:
Yea if you're targeting `netcoreapp3.0` and using the `Microsoft.NET.Sdk.Web` SDK then the framework reference is implicitly added for you. Since this is just using the "Microsot.NET.Sdk" SDK (for libraries) then we need to explicitly add it, otherwise, it's the same answer:
> we need stuff that was previously in Microsoft.AspNetCore.Hosting.Abstractions and is now only available in the shared framework.